iOS/Flows/People/PeopleCoordinator.swift

```diff
@@ -15,7 +15,7 @@ import Localization
 
 class PeopleCoordinator: PresentableCoordinator<[Person]> {
 
-    private lazy var peopleNavController = PeopleNavigationController()
+    private lazy var peopleNavController = PeopleNavigationController(showConnections: self.selectedConversationCID.exists)
```

Comment:
No reason to show connections when you are just inviting people without a conversation. I tried thinking through the UX of inviting people and then adding them to conversations, but it just didn't make sense. Nor did it make sense to create one, then have everyone you invited be added to it. Maybe there is a better way...